sql/augments.sql

```diff
@@ -1343,68 +1348,68 @@ INSERT INTO `augments` VALUES (1077,5,779,1,0,0); -- Split Shot:DMG+5% (increase
+INSERT INTO `augments` VALUES (1095,5,741,1,0,0); -- Mystic Boon:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1096,5,627,1,0,0); -- Scourge:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1097,5,629,1,0,0); -- Torcleaver:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1098,5,643,1,0,0); -- Onslaught:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1099,5,644,1,0,0); -- Primal Rend:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1100,5,646,1,0,0); -- Cloudsplitter:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1101,5,659,1,0,0); -- Metatron Torment:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1102,5,660,1,0,0); -- King's Justice:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1103,5,662,1,0,0); -- Ukko's Fury:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1104,5,675,1,0,0); -- Catastrophe:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1105,5,676,1,0,0); -- Insurgency:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1106,5,678,1,0,0); -- Quietus:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1107,5,691,1,0,0); -- Geirskogul:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1108,5,692,1,0,0); -- Drakesbane:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1109,5,694,1,0,0); -- Camlann's Torment:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1110,5,707,1,0,0); -- Blade: Metsu:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1111,5,708,1,0,0); -- Blade: Kamu:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1112,5,710,1,0,0); -- Blade: Hi:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1113,5,723,1,0,0); -- Tachi: Kaiten:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1114,5,724,1,0,0); -- Tachi: Rana:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1115,5,726,1,0,0); -- Tachi: Fudo:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1116,5,755,1,0,0); -- Gate of Tartarus:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1117,5,758,1,0,0); -- Omniscience:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1118,5,756,1,0,0); -- Vidohunir:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1119,5,757,1,0,0); -- Garland of Bliss:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1120,5,770,1,0,0); -- Namas Arrow:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1121,5,702,1,0,0); -- Jishnu's Radiance:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1122,5,787,1,0,0); -- Trueflight:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1123,5,786,1,0,0); -- Coronach:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1124,5,788,1,0,0); -- Leaden Salute:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1125,5,790,1,0,0); -- Wildfire:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1126,5,631,1,0,0); -- Dimidiation:DMG+5% (increases by 5)
+INSERT INTO `augments` VALUES (1127,5,0,1,0,0); -- Exudation:DMG+5% (increases by 5)
```

Comment:
Exudation isn't implemented yet, but the ID will 175 be it gets implemented. So this should be 745 (+1 of Realmrazer, just before Heavy Swing)